CorsixTH/Lua/hospital.lua

```diff
@@ -664,7 +664,7 @@ function Hospital:countPatients()
   -- i.e. calling it from town map to use here
   -- so Town map now takes this information from here.  (If I am wrong, put it back)
   self.patientcount = 0
-  for _, patient in pairs(self.patients) do
+  for _, patient in ipairs(self.patients) do
```

Comment:
Other places use it as a list (which is stupid imho, but I am doing patient rather than hospital atm).